web_widget_color_tree_field/README.rst

```diff
@@ -0,0 +1,80 @@
+Colorize field in tree views
+============================
+
+This module aims to add support for dynamically coloring fields in tree view
+according to data in the record.
+
+It provides new attributes with the same syntax as 'colors' attribute in tree tag.
+
+Features
+========
+
+* Add attribute 'bg_color' to color background of a cell in tree view
+
+* Add attribute 'fg_color' to change text color of a cell in tree view
+
+
+Usage
+=====
+
+* In the tree view declaration, put bg_color="red:customer==True;" attribute in the field tag::
+
```

Comment:
I forgot to mention that :
   - after installing : click on the enterprise/Odoo logo to reload javascript files
   - fields mentionned in attributes must be present in tree view (even in invisible). This is the same behaviour as colors in tree tag.